tests/unit/redux-model-test.js

```diff
@@ -135,21 +125,25 @@ describe('Redux Models', () => {
   });
 
   describe('with incomplete data', () => {
-    beforeEach(() => {
-      // this will produce an empty request object
-      let vendorRequest = resolver.getRequest('find', { type: 'vendors' });
-      let pkgRequest = resolver.getRequest('find', { type: 'packages' });
+    let INCOMPLETE_VENDOR_JSON = {
+      ...VENDOR_JSON,
+      attributes: { totalPackages: 1 },
+      relationships: { packages: {} }
+    };
 
-      vendor = new VendorModel({
-        ...VENDOR_JSON,
-        attributes: { totalPackages: 1 },
-        relationships: { packages: {} }
-      }, vendorRequest, resolver);
+    let INCOMPLETE_PACKAGE_JSON = {
```

Comment:
why `let` and then CONSTANT_CASE?

Reply:
I'm already trained to only use `const` only in the outer scope, so I accidentally `let` these in.